test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/npd-binding.yaml

```diff
@@ -0,0 +1,16 @@
+# This is the role binding for the node-problem-detector
+# which is used to get and patch the node health status.
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: node-problem-detector-binding
+  labels:
+    kubernetes.io/cluster-service: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
NPD is only reading and modifying nodes (or to be more specific I think modifying only node status). I think we should make a dedicated role for it and use it here.
@deads2k - do you agree with it? This will most probably be a very simple role.
@shyamjvs - if David agrees, can you make that PR?